[2025-10-11T07:44:14.395+0200] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: predict_batch_dag.send_batch_predictions scheduled__2025-10-11T05:22:35+00:00 [queued]>
[2025-10-11T07:44:14.397+0200] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: predict_batch_dag.send_batch_predictions scheduled__2025-10-11T05:22:35+00:00 [queued]>
[2025-10-11T07:44:14.398+0200] {taskinstance.py:2193} INFO - Starting attempt 4 of 1
[2025-10-11T07:44:14.402+0200] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): send_batch_predictions> on 2025-10-11 05:22:35+00:00
[2025-10-11T07:44:14.405+0200] {standard_task_runner.py:60} INFO - Started process 48121 to run task
[2025-10-11T07:44:14.408+0200] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'predict_batch_dag', 'send_batch_predictions', 'scheduled__2025-10-11T05:22:35+00:00', '--job-id', '247', '--raw', '--subdir', 'DAGS_FOLDER/prediction_dag.py', '--cfg-path', '/var/folders/rp/syznybx160jd0kg9k7r_sj8w0000gn/T/tmp6jmeq718']
[2025-10-11T07:44:14.409+0200] {standard_task_runner.py:88} INFO - Job 247: Subtask send_batch_predictions
[2025-10-11T07:44:14.427+0200] {task_command.py:423} INFO - Running <TaskInstance: predict_batch_dag.send_batch_predictions scheduled__2025-10-11T05:22:35+00:00 [running]> on host ebots-macbook-pro.local
[2025-10-11T07:44:14.447+0200] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='predict_batch_dag' AIRFLOW_CTX_TASK_ID='send_batch_predictions' AIRFLOW_CTX_EXECUTION_DATE='2025-10-11T05:22:35+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-11T05:22:35+00:00'
[2025-10-11T07:44:14.448+0200] {prediction_dag.py:41} INFO - ðŸ“„ Found CSV file: /Users/ebotfabien/Desktop/school/air_new/dags/destination/data_part_17.csv
[2025-10-11T07:44:14.451+0200] {prediction_dag.py:68} INFO - ðŸ“¦ Sending 10 batch(es)...
[2025-10-11T07:44:14.452+0200] {prediction_dag.py:73} INFO - ðŸ“¦ Sending batch 1/10
[2025-10-11T07:47:48.298+0200] {local_task_job_runner.py:121} ERROR - Received SIGTERM. Terminating subprocesses
[2025-10-11T07:47:48.308+0200] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 48121. PIDs of all processes in the group: [48121]
[2025-10-11T07:47:48.309+0200] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 48121
[2025-10-11T07:47:53.068+0200] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: predict_batch_dag.send_batch_predictions scheduled__2025-10-11T05:22:35+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2025-10-11T07:47:53.071+0200] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: predict_batch_dag.send_batch_predictions scheduled__2025-10-11T05:22:35+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2025-10-11T07:47:53.071+0200] {local_task_job_runner.py:160} INFO - Task is not able to be run
